### PR TITLE
Prepare release for 1.8.4

### PR DESCRIPTION
**Pull request by ConnorMcMahon**
- Update to version 1.8.4
- Use DurableTask.AzureStorage to 1.6.5
- Update samples to use new version.

### CODE REVIEW

**Comment by ConnorMcMahon** [comment]
There are 3 open issues on the milestone that aren't `fix-ready` or `documentation` tagged.

#771 - I'm hesitant to move forward on this issue until a non-prerelease of Microsoft.NET.Sdk.Functions that unblock 2.3.0 usage moves forward.

#951 - Not sure how much work needs to be done on the extension side to light this up. Will investigate.

#934 - Not sure if this is something we can address until we support BYO serialization, which is likely a 2.0 only feature. The customer has a workaround, so I don't necessarily want to hold on the release for this.

**Comment by ConnorMcMahon** [comment]
I pinged Fabio in one of the core open issues in the corresponding repo, so I will know soon regarding that issue.

File: src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj [hunk 1]
```diff
@@ -45,7 +45,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.3" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.4" />
```
Comment (amdeel) [inline src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj]: Is the intention to update to the newest version, 1.6.5?

**Comment by amdeel** [comment]
The #934 fix would probably include custom serialization settings in DurableTask.Core, the changes that I am currently working on. I can prioritize this work if it would help.

File: src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj (hunk 1 above)
Comment (ConnorMcMahon) [reply src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj]: Good call!

**Pull request merged**